proxy/proxy/spacewalk-proxy.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Fri Nov 12 14:11:49 CET 2021
+
```

Comment:
This is a bit special in this git repo that we do not need the "header" for the changelog.
The header will be added automatically when we submit the package.
```suggestion
```